### PR TITLE
Clown cake can now be sliced.

### PR DESCRIPTION
**Pull request by Mitchs98**
## What Does This PR Do
Let there be cake. Clown cake can be sliced now. Whoops. Taste description properly modified.

## Why It's Good For The Game
There's a clown cake slice, it should be obtainable. Banana does not taste like cake.

## Testing
I cut a cake :D

## Changelog
:cl:
fix: You can now slice clown cake.
fix: Clown cake now tastes correctly.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/food/foods/frozen.dm [hunk 1]
```diff
@@ -17,6 +17,8 @@
 	icon = 'icons/obj/food/frozen_treats.dmi'
 	icon_state = "clowncake"
 	slice_path = /obj/item/reagent_containers/food/snacks/frozen/clowncakeslice
+	slices_num = 5
+	bitesize = 3
 	list_reagents = list("nutriment" = 20, "sugar" = 5, "vitamin" = 5, "banana" = 15)
 	tastes = list("cake" = 5, "sweetness" = 2, "carrot" = 1)
```
Comment (Henri215) [inline code/modules/food_and_drinks/food/foods/frozen.dm:23]: carrot is intended?

Comment (Mitchs98) [reply code/modules/food_and_drinks/food/foods/frozen.dm:23]: ...You see bananas magically morph into tasting like carrots.

Yeah I didn't notice that and it wasn't a recent change I just wanted the cake to slice lol.

Comment (Henri215) [reply code/modules/food_and_drinks/food/foods/frozen.dm]: Hey, its clown magic :)

**Review by Henri215 — APPROVED** [review]
LHTM

File: code/modules/food_and_drinks/food/foods/frozen.dm [hunk 2]
```diff
@@ -17,8 +17,10 @@
 	icon = 'icons/obj/food/frozen_treats.dmi'
 	icon_state = "clowncake"
 	slice_path = /obj/item/reagent_containers/food/snacks/frozen/clowncakeslice
+	slices_num = 5
+	bitesize = 3
 	list_reagents = list("nutriment" = 20, "sugar" = 5, "vitamin" = 5, "banana" = 15)
-	tastes = list("cake" = 5, "sweetness" = 2, "carrot" = 1)
```
Comment (Contrabang) [inline code/modules/food_and_drinks/food/foods/frozen.dm:21]: mention this in the PR as part of the fix, otherwise lgtm

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by DGamerL — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**